fmt(cstyle): do not validate src/config.h

marcoandredinis opened the pull request:
This file is auto-generated so it should not be validated when running `make
cstyle`.

You can find out more information here: 
https://github.com/openebs/openebs/issues/2009#issuecomment-427594728

marcoandredinis committed:
fmt(cstyle): do not validate src/config.h

This file is auto-generated so it should not be validated when running `make
cstyle`.

utkarshmani1997 reviewed (approved):
LGTM

vishnuitta commented on Makefile.in:
```diff
@@ -71,6 +71,7 @@ distdir:
 
 cstyle:
 	find $(top_srcdir) -name '*.[hc]' \
+                ! -wholename './src/config.h' \
```
can you please change `wholename` to `path`, being more portable?

marcoandredinis replied:
Sure!

vishnuitta reviewed (changes requested):
changes as requested in comments please

marcoandredinis committed:
Use `path` instead of `wholename`

The former is more portable

https://www.gnu.org/software/findutils/manual/html_node/find_html/Full-Name-Patterns.html

vishnuitta reviewed (approved):
changes are good

vishnuitta committed:
Merge branch 'replication' into ignore-style-config
